bot/utilities/Queue.ts

```diff
@@ -36,6 +37,14 @@ export default class Queue {
     return this.entries.length;
   }
 
+  isEmpty(): boolean {
+    return this.length == 0;
+  }
+
```

Comment:
It seems a little curious to have both an `isEmpty` method and an `isNotEmpty` method; `isNotEmpty` returns the negation of `isEmpty`, so the user can just use the value `!Queue.isEmpty`.

Reply:
Good point. I debated the same thing and chose to err on the side of explicitness in this case. It would make sense for an interface being more widely interacted with/reused. Removed!